sdcm/nemesis.py

```diff
@@ -656,7 +657,7 @@ def disrupt_truncate(self):
         # do the actual truncation
         self.target_node.run_cqlsh(cmd='TRUNCATE {}.{}'.format(keyspace_truncate, table), timeout=120)
 
-    def _modify_table_property(self, name, val, filter_out_table_with_counter=False):
+    def _modify_table_property(self, name, val, filter_out_table_with_counter=False, modify_all_tables=False):
         disruption_name = "".join([p.strip().capitalize() for p in name.split("_")])
```

Comment:
@yarongilor   modify_all_tables argument is unused.

Reply:
That's right, but it may be useful to add support for it in the future. if it causes pylint issue, then it better be removed indeed.